### PR TITLE
Refactoringiossdk

### PR DESCRIPTION
**Pull request by himanshuatjudo**

### CODE REVIEW

File: samples/iOS/JudoPayiOSXamarinSampleApp/AppDelegate.cs [hunk 1]
```diff
@@ -60,13 +60,7 @@ void ConfigureJudoSettings ()
...
 			configInstance.ApiToken = "5tZfrXDngpvu8iGS";
 			configInstance.ApiSecret = "da36e4c8f5805173060c934b12dcc14bb05761af310ea364cd787710b1da346b";
-			configInstance.JudoId = "100515592";//100515-592
-			configInstance.PaymentReference = "payment101010102";
-			configInstance.ConsumerRef = "consumer1010102";
-
-			JudoSDKManager.UIMode = true;
-            JudoSDKManager.RiskSignals = true;
-			JudoSDKManager.MaestroAccepted = true;
+			configInstance.JudoId = "100515592";
```
Comment (hirenhirani) [inline samples/iOS/JudoPayiOSXamarinSampleApp/AppDelegate.cs]: Test Comment (Ignore this)

**Comment by hirenhirani** [comment]
+1

**Comment by NickAllport** [comment]
:+1: approved

**Comment by NickAllport** [comment]
+1 Approve

**Pull request merged**